cmd/Kinsokujiko/main.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo"
+	"github.com/sys-cat/Kinsokujiko"
+)
+
+type Kinsoku struct {
```

Comment:
__[golint]__
* [golint] exported type Kinsoku should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->